### DISCUSSION TITLE
Cara mengembalikan/restore Environtment yang terhapus

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#604] [Q&A] [answered]
Halo kak permisi saya ingin bertanya terkait cara mengembalikan environtment yang terhapus, jadi waktu itu saya membuat environtment dengan conda create -p path saya(jadi tidak ada nama environtment hanya langsung ke path filenya saja), lalu saya tidak sengaja mengubah nama folder saya, ketika saya mengecek env list saya tiba tiba file tersebut hilang, namun ketika saya ingin membuat baru di path yang sama muncul seperti ini 
![image](https://user-images.githubusercontent.com/78478403/228562565-87bed3da-917e-4f5f-8d8b-97aaf13c857c.png)

tapi ketika mengecek conda info --envs malah tidak muncul 
![image](https://user-images.githubusercontent.com/78478403/228562610-6be75390-655b-4ad8-8057-1e38bbbd3425.png)

Apakah ada cara untuk mengembalikan environtment tersebut? atau harus membuat dari awal lagi

Terima kasih

**Comment by Yusraf002** [comment, marked as answer]
Dear @daryraa 
Terima kasih atas pertanyaannya Pak, berdasarkan gambar pertama dan gambar kedua
solusi terbaik yg dapat kami sarankan adalah membuat ulang environtment Bapak dan agar hal tersebut tidak terjadi kembali, harap tidak melupakan pemberian nama pada environtment yang Bapak buat.

Demikian yang dapat kami sampaikan. Semoga membantu!

Best regards,
Team Algoritma

**Reply by daryraa** [reply]
baik pak terima kasih
